docs/docs/setting-up/networking-instructions/networking.md

```diff
@@ -3,25 +3,29 @@ sidebar_position: 4
 sidebar_label: "Internet Access"
 ---
 
-# Accessing the Internet from Jobs
+# Internet access
```

Comment:
I wonder if we need to rewrite this more significantly. There's nothing fundamentally wrong with it, but it's framed in a weird way. You can use external docker containers, for example. It's just external network access that is limited - not necessarily to the internet, even to other nodes on the same network.

Further, the `--network` tag needs to be paired with a server with similar flags - do we need to detail that?

Reply:
Could you please provide more details for the node/server flag pairing? Seems this is not currently reflected in the documentation

Reply:
If a user specifies the --network flag, then it will only be run on compute nodes that were started with the `--job-selection-accept-networked ` flag. If no compute nodes were started with that flag, the job will not run.

Reply:
Added a note for this case